### DISCUSSION TITLE
Setup Help

### DISCUSSION THREAD

**Discussion by taurus042898** [chuot/rdio-scanner#363] [Q&A]
Hello, i was reaching out because i am having issues setting this up. I live in Ohio in the Akron area, and want to have it play for this area. the scanner i usually use isn't always up especially when something big is going down because the person didn't have it up on broadcastify. Any pointers would definitely be helpful. I'm currently running on Linux arch for the computer, and have a iPhone XS max running the ios 17 os.

**Comment by repairfox** [comment]
@taurus042898, do you still need help?
